### ISSUE TITLE
MAUI BlazorWebView is zoomed out on Mac

### ISSUE THREAD

**Issue by veler** [dotnet/maui#12829]
### Description

Side by side view of a blank Blazor Web App running in Safari, and a blank MAUI Blazor app.

![image](https://user-images.githubusercontent.com/3747805/213955890-d0933d63-6c43-43d8-adcb-71d4ef83522a.png)

The MAUI app is zoomed out, making the text and other UI element smaller and therefore harder to read.

### Steps to Reproduce

1. Create a File > New .NET MAUI Blazor App
2. Build and Run the Mac-Catalyst app.

**Expected behavior:**
An app with a BlazorWebView at 100% zoom.

**Current behavior:**
It is zoomed out, even if in `Index.html`, I have:
`<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no, viewport-fit=cover" />`
See screenshot above.

If I set Safari zoom to 90%, then I get the same thing than in MAUI. So it means MAUI BlazorWebView is likely at 90% zoom for some reasons. But when looking at the WKWebView in the debugger, I can see that PageZoom is 1 (meaning 100%). So I'm not sure to understand what's going on.
![image](https://user-images.githubusercontent.com/3747805/213970645-d485ffe7-bf68-453c-ba2b-509ac290fe2f.png)

I can reproduce the issue with .NET 6 and .NET 7.
I do not reproduce the issue on Windows.
I did not try another platform.

I use a Mac Mini M1, macOS Ventura 13.1. My monitor is a BenQ GW2480 24, 1080p.
This is the only app on my Mac with this issue. Safari and Edge don't have this problem.

### Link to public reproduction project repository

N/A

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

macOS

### Affected platform versions

macOS 13.1

### Did you find any workaround?

No. Please provide one if possible.

**Comment by veler** [comment]
Alright, I found the reason of this issue!

Here is a related issue I found:
https://github.com/dotnet/maui/issues/10622

For whoever come across this same issue, according to the following: https://github.com/xamarin/xamarin-macios/issues/14621

> When developing an app with MAUI and targeting net6.0-maccatalyst, the toolchain will currently default to what Xcode's UI calls "Scale Interface to Match iPad". This will scale the interface to 77%.

The solution is to edit the `Info.plist` file in the Csproj and set `UIDeviceFamily` to 6. Here is an example:
```
    <key>UIDeviceFamily</key>
    <array>
        <!-- Targets iPhone -->
        <integer>1</integer>
        <!-- Targets iPad -->
        <integer>2</integer>
        <!-- Targets macOS -->
        <integer>6</integer>
    </array>
```

Note that doing so seem to risk to break other MAUI components, like Button. It sounds like MAUI aims at supporting iOS / iPad well but not so well macOS. I can understand that considering there's likely a bigger market to reach on smartphone / tablet than desktop, but it's quite frustrating that desktop suffers from this kind of issue out of the box.

Feel free to mark this issue as duplicated and close it, or keep it open if you think it would make sense to do so.
